lib/go-atscfg/hostingdotconfig.go

```diff
@@ -118,15 +118,18 @@ func MakeHostingDotConfig(
 
 	filteredDSes := []DeliveryService{}
 	for _, ds := range deliveryServices {
-		if ds.Active == nil || ds.Type == nil || ds.XMLID == nil || ds.CDNID == nil || ds.ID == nil || ds.OrgServerFQDN == nil {
+		if &ds.Active == nil || ds.Active == "" || ds.Type == nil || &ds.XMLID == nil || &ds.CDNID == nil || ds.ID == nil || ds.OrgServerFQDN == nil {
```

Comment:
No need to check `&ds.Active == nil`, `ds.Active` is a value and not a pointer.

Reply:
fixed